packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/command/translations/Messages_fr_FR.json

```
@@ -0,0 +1,22 @@
+{
+  "pci_databases_fork_command_close_button": "Fermer",
+  "pci_databases_fork_command_title": "Commande équivalente",
+  "pci_databases_fork_command_description": "La commande suivante vous permet de créer votre service grâce à notre API.",
+  "pci_databases_fork_command_description_terraform": "La resource suivante vous permet de créer votre service grâce à notre Terraform OVH Provider.",
```

```suggestion
  "pci_databases_fork_command_description_terraform": "La resource suivante vous permet de créer votre service grâce à notre Terraform OVHcloud Provider.",
```
I don't know if it's marketing name or if you can change OVH by OVHcloud